fabfile.py

```diff
@@ -407,6 +407,29 @@ def workload(c, upload=None, describe=None):
     
     return current_workload or None
 
+# The function will take a specified snapshot_id and delete the corresponding snapshot.
+@task
+def snapshot_delete(c, snapshot_id=None):
```

Comment:
Please make the arg mandatory.
```suggestion
def snapshot_delete(c, snapshot_id):
```
You can then remove the code just below.

Reply:
This allows me to give a more descriptive error message

Reply:
As discussed, let's leave it.